src/components/tools/text-toolbar-button.tsx

```diff
@@ -9,19 +18,32 @@ interface IProps {
   onClick: (e: React.MouseEvent) => void;
 }
 
-const iconClasses = (iconName: string, enabled: boolean, isSelected: boolean) => {
-  return classNames("button-icon", "fa", "fa-fw", `fa-${iconName}`, isSelected ? "on" : "off", { enabled });
+const kTooltipYDistance = 2;
+const buttonIcon = (iconName: string, enabled: boolean, isSelected: boolean, onClick: (e: React.MouseEvent)=>void) => {
+  const iconClasses = classNames("button-icon", isSelected ? "on" : "off", { enabled });
+  const iconNameArr = ["bold", "italic", "underline", "superscript", "subscript", "list-ol", "list-ul"];
```

Comment:
Shouldn't need to redundantly specify the list of icon names. Just incorporate the additional information you need into the `buttonDefs` structure in the `<TextToolbarComponent>`.